Changing Set-AzureSqlDatabaseAuditingPolicy to Set-AzureSqlDatabaseAu…

SvenAelterman opened the pull request:
…diting

Per output of Set-AzureSqlDatabaseAuditingPolicy, it is obsolete now.
I would use this new cmdlet also in the script, but I am unable to locate the GitHub repo where the sample script lives 😟.

Also, changed hard-coded resource group name in Remove cmdlet to the variable value. The resource group name has random values in it.

azurecla commented:
Hi __@sachouks__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (sachouks). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

SvenAelterman committed:
Changing Set-AzureSqlDatabaseAuditingPolicy to Set-AzureSqlDatabaseAuditing

Per output of Set-AzureSqlDatabaseAuditingPolicy, it is obsolete now.
I would use this new cmdlet also in the script, but I am unable to locate the GitHub repo where the sample script lives 😟.

Also, changed hard-coded resource group name in Remove cmdlet to the variable value. The resource group name has random values in it.
